ocs_ci/ocs/resources/pod.py

```diff
@@ -745,3 +746,21 @@ def delete_pods(pod_objs):
     for pod in pod_objs:
         pod.delete()
     return True
+
+
+def get_used_space_on_mount_point(pod_obj):
+    """
+    Get the used space on a mount point
+
+    Args:
+        pod_obj (POD): The pod object
+
+    Returns:
+        int: Percentage represent the used space on the mount point
+
+    """
+    # Verify data's are written to mount-point
+    mount_point = pod_obj.exec_cmd_on_pod(command="df -kh")
+    mount_point = mount_point.split()
+    used_percentage = mount_point[mount_point.index('/var/lib/www/html') - 1]
```

Comment:
please create a constant for /var/lib/www/html

Reply:
done